src/containers/ManageListingsPage/ManageListingsPage.js

```diff
@@ -165,70 +240,23 @@ export const ManageListingsPageComponent = props => {
 
         <div className={css.listingPanel}>
           <Heading listingsAreLoaded={listingsAreLoaded} pagination={pagination} />
-
-          <div className={css.listingCards}>
-            {listings.map(l => (
-              <ManageListingCard
-                className={css.listingCard}
-                key={l.id.uuid}
-                listing={l}
-                isMenuOpen={!!listingMenuOpen && listingMenuOpen.id.uuid === l.id.uuid}
-                actionsInProgressListingId={openingListing || closingListing}
-                onToggleMenu={onToggleMenu}
-                onCloseListing={onCloseListing}
-                onOpenListing={handleOpenListing}
-                hasOpeningError={openingErrorListingId.uuid === l.id.uuid}
-                hasClosingError={closingErrorListingId.uuid === l.id.uuid}
-                renderSizes={renderSizes}
-              />
-            ))}
+          <div className={css.listingCardsTabs}>
+            <Tabs
+              defaultActiveKey={currentListingType}
+              onChange={onTabChange}
+              items={[
+                { key: 'product-listing', label: 'Shop', children: listingRenderer },
```

Comment:
Don't use the types directly as string, instead when possible use this:
LISTING_TYPES
marketplace/src/util/types.js
marketplace/server/api-util/metadataHelper.js
(depending on if you are on the FE or the BE)